src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala

```diff
@@ -52,82 +129,197 @@ import scala.collection.mutable.ListBuffer
+          val rawCounts = BaseCounts(rawByMiAndReadNum(mi + readNumberSuffix(c.getRecord)))
+
+          val m =  ConsensusVariantReviewInfo(
+            chrom = variant.chrom,
+            pos   = variant.start,
+            ref   = variant.refBase.toString,
+            genotype = variant.genotype.getOrElse("NA"),
+            filters  = variant.filters.getOrElse(""),
+            A = consensusCounts.a,
+            C = consensusCounts.c,
+            G = consensusCounts.g,
+            T = consensusCounts.t,
+            N = consensusCounts.n,
+            consensus_read = consensusReadName,
+            consensus_insert = toInsertString(c.getRecord),
+            consensus_call = c.getReadBase.toString.toUpperCase,
+            consensus_qual = c.getBaseQuality,
+            a = rawCounts.a,
+            c = rawCounts.c,
+            g = rawCounts.g,
+            t = rawCounts.t,
+            n = rawCounts.n
+          )
+
+          writer.write(m)
+        }
+    }
+
+    writer.close()
   }
 
   /** Loads the set of locations for review. */
-  private[umi] def loadPositions(path: FilePath, refFile: ReferenceSequenceFile): Seq[LocAndRef] = {
-    val buffer = ListBuffer[LocAndRef]()
+  private[umi] def loadPositions(path: FilePath, refFile: ReferenceSequenceFile): Seq[Variant] = {
+    val buffer = ListBuffer[Variant]()
 
     if (VcfExtensions.exists(ext => path.getFileName.toString.endsWith(ext))) {
       val in = new VCFFileReader(input.toFile, false)
-      in.filter(_.isSNP).foreach(v => buffer += LocAndRef(v.getContig, v.getStart, v.getReference.getBases()(0).toChar.toUpper))
+      val sample = this.sample.orElse {
+        if (in.getFileHeader.getSampleNamesInOrder.size() == 1) Some(in.getFileHeader.getSampleNamesInOrder.get(0))
+        else None
+      }
+
+      in.filter(_.isSNP).filter(v => mafFromGenotype(v, sample).forall(_ <= this.maf)).foreach { v => buffer += Variant(
+        chrom    = v.getContig,
+        start    = v.getStart,
+        refBase  = v.getReference.getBases()(0).toChar.toUpper,
+        genotype = sample.map(v.getGenotype(_).getGenotypeString),
+        filters  = Some(v.getFilters.toSeq.sorted.mkString(","))
+      )}
     }
     else {
       val list = IntervalList.fromFile(path.toFile).uniqued(false)
       for (i <- list; pos <- i.getStart to i.getEnd) {
-        buffer += LocAndRef(i.getContig, pos, refFile.getSubsequenceAt(i.getContig, pos, pos).getBases()(0).toChar.toUpper)
+        buffer += Variant(i.getContig, pos, refFile.getSubsequenceAt(i.getContig, pos, pos).getBases()(0).toChar.toUpper, None, None)
       }
     }
 
     buffer.toList
   }
 
+  /** Attempts to determine the minor allele fraction from a VCF genotype object. */
+  private def mafFromGenotype(ctx: VariantContext, sample: Option[String]): Option[Double] = sample match {
+    case None    => None
+    case Some(s) =>
+      val gt = ctx.getGenotype(s)
+      Option(gt.getExtendedAttribute("AF")) match {
```

Comment:
I'm honestly not sure what the right thing to do here is.  It's _not_ a field that is standardized in VCF, so some tools use `AF` others use other fields.  `AlleleFractionField` in `MakeMixtureVcf` represents the choice in that tool to _write_ a field called `AF`, but note that it supports the user's choice of input field.  Freakin VCF.